R/metab_simple.R

```diff
@@ -24,7 +24,9 @@
 #'   
 #' @author Alison Appling, Jordan Read; modeled on lakeMetabolizer
 #' @examples
-#'  metab_simple(data=data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, DO.sat=rep(14, 24), PAR=sin((1:24)*pi/24)^8, temp.water=15))
+#'  metab_simple(data=data.frame(
```

Comment:
do.sat was passed in for LakeMetabolizer because the data and metadata requirements to calculate it include information you might not normally pass in. Model choice, elevation or barometric pressure. Making people calculate it ahead of time adds tasks for the user but reduces data inputs.